README.md

```diff
@@ -87,6 +87,20 @@ Read more about [how KRR works](#how-it-works) and [KRR vs Kubernetes VPA](#diff
 
 ## Installation
 
+### Requirements
+
+KRR requires you to have Prometheus, running either on your cluster or centralized.
+
+Additionally to that, [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) needs to be running on your cluster, as KRR is dependant on next metrics:
+
+- `container_cpu_usage_seconds_total`
+- `container_memory_working_set_bytes`
+- `kube_replicaset_owner`
+- `kube_pod_owner`
+- `kube_pod_status_phase`
+
+_Note: If one of last three metrics is absent KRR will still work, but will result in potentially worse results and raise a warning_
```

Comment:
Can you explain why they will be worse?

Reply:
It is explained in a PR description:

>Fallback to loading pods from KubeAPI in case some error occured with prometheus.
Raise a warning, as this results in missing some pods from the total calculation.

>Note: previously (<= 1.4) KRR was loading pods from the API, and then loading deleted ones from prometheus (as only prometheus keeps track of historical objects). 1.5 introduced an optimization of loading everything only from prometheus, but some users (https://github.com/robusta-dev/krr/issues/127#issuecomment-1737223230) seem to not have some required metrics.

Reply:
Yes, but it will be missing for future readers of the README who don't know anything about this PR.

Assuming I understood correctly, I suggest:

_Note: If one of last three metrics is absent KRR will still work, but it will only consider currently-running pods when calculating recommendations. Historic pods that no longer exist in the cluster will not be taken into consideration._